src/app/registration/facility-select/facility-select.component.ts

```diff
@@ -261,6 +272,21 @@ export class FacilitySelectComponent implements OnInit {
     return DateTime.now().setZone('America/Vancouver');
   }
 
+  async getSystemTime(){
```

Comment:
spacing

Reply:
Resolved in [Commit 43beab5](https://github.com/bcgov/parks-reso-public/pull/461/commits/43beab52a1c7d818910d777c2133aefa47ce60ff)